File: 08-backbone/blog_rails_app/app/assets/javascripts/backbone/models/Comment.js
var blogApp = blogApp || {};

blogApp.Comment = Backbone.Model.extend({
  urlRoot: '/comments',
  schema: {
    post_id: 'Hidden',
    author: 'Text',
    blather: 'TextArea'
  },
  defaults: {
    post_id: null,
    author: '',
    blather: ''
  }
});

File: 08-backbone/blog_rails_app/app/models/comment.rb
# == Schema Information
#
# Table name: comments
#
#  id         :integer          not null, primary key
#  post_id    :integer
#  author     :string(255)
#  blather    :text
#  created_at :datetime         not null
#  updated_at :datetime         not null
#

class Comment < ActiveRecord::Base
  attr_accessible :post_id, :author, :blather
  belongs_to :post
end
